package.json

```diff
@@ -18,7 +18,7 @@
     "url": "https://github.com/prismatic-io/embedded.git"
   },
   "license": "MIT",
-  "version": "2.8.0",
+  "version": "2.8.1",
```

Comment:
This should be `2.9.0` so its net new functionality.

Reply:
Good call; I'll bump it to 2.9.0 :thumbsup: